src/test/pool_test/test_ccmp.cc

```diff
@@ -26,10 +26,13 @@ class YccmpTest: public CppUnit::TestFixture {
 
   CPPUNIT_TEST_SUITE(YccmpTest);
    
+#ifdef __linux__
+// uses /proc, does not exist on OSX
```

Comment:
Shouldn't this be #ifndef _DARWIN_ or something.
There should be some sort of /proc on the later bsd's.  (I don't have a later bsd handy(